Arduino/Propulsion/src/PropulsionModule.cpp

```diff
@@ -35,7 +41,21 @@ uint16_t byteToShort(int16_t index);
 void errorReply(uint8_t errorByte);
 
 /*****Propulsion Module Specific Parameters**************************/
+#ifdef FEATHER
+Adafruit_MotorShield motorShield = Adafruit_MotorShield();
+Adafruit_DCMotor *m0 = motorShield.getMotor(3);
+Adafruit_DCMotor *m1 = motorShield.getMotor(4);
+void initDriver() {
```

Comment:
We're invoking `initDriver` twice, once for each condition. Could we have the function call prior to or after the conditional block?

Reply:
Which invocations are you talking about?